sotodlib/tod_ops/azss.py

```diff
@@ -24,11 +26,21 @@ def bin_by_az(aman, signal=None, az=None, range=None, bins=100, flags=None):
     range: array-like, optional
         A list specifying the range of azimuth angles to consider for binning. Defaults to None.
         If None, [min(az), max(az)] will be used for binning.
-    bins: integer
-        The number of bins to divide the azimuth angle range into. Defaults to 360.
+    bins: int or sequence of scalars
+        If bins is an int, it defines the number of equal-width bins in the given range (100, by default).
+        If bins is a sequence, it defines the bin edges, including the rightmost edge, allowing for non-uniform bin widths.
+        If `bins` is a sequence, `bins` overwrite `range`.
```

Comment:
In docs/docstrings you need double backwards commas: "``" instead of "`" to get the text to render as you want.